packages/edge-functions/dev/node/main.ts

```diff
@@ -1,7 +1,7 @@
 import path from 'node:path'
 import { fileURLToPath, pathToFileURL } from 'node:url'
 
-import { renderFunctionErrorPage, type Geolocation } from '@netlify/dev-utils'
+import { Logger, renderFunctionErrorPage, type Geolocation } from '@netlify/dev-utils'
```

Comment:
We should just enable [`verbatimModuleSyntax`](https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax) so this is automatic. I'll stop pointing these out for now. 😄

```suggestion
import { type Logger, renderFunctionErrorPage, type Geolocation } from '@netlify/dev-utils'
```